### PR TITLE
Game should be able to load now

### PR DESCRIPTION
**Pull request by kpugington**

### CODE REVIEW

**Comment by kazyofloof** [comment]
Currently broken when playing with this fork, from what I've discovered:
The "findable" is being disregarded, at least for modded symbols.
~~The new floors do not appear.~~ They will if re-added after the first time the modloader loads, it may have been an issue with the value being in my LBAL-LuckyAPI.save. I am unsure if they will be unlocked as per usual. Because the starting duds are added by the modloader instead of using the game's system, floors with more than two duds do not add more duds properly. The "dud every x rounds" effect may need investigation.
~~When continuing a save, items are loaded from LBAL.save rather than LBAL-LuckyAPI.save~~ This only happened to me on the first run, and I'm not entirely sure why. 
Somewhat surprisingly, comfy pillow did function.

File: luckyapi/modloader/patches/Pop-up.gd [hunk 1]
```diff
@@ -5,132 +5,6 @@ onready var modloader: Reference = get_tree().modloader
 func option_is_findable(type):
     return modloader.can_find_symbol(type)
 
-func add_cards(f_rarities):
```
Comment (kazyofloof) [inline luckyapi/modloader/patches/Pop-up.gd:8]: Although this does make it possible to start a run, it isn't a solution; this code is here for a reason. Its removal has likely broken multiple things, but at minimum it makes modded symbols that are marked as "unfindable"— only obtainable through interactions— findable.

Comment (kpugington) [reply luckyapi/modloader/patches/Pop-up.gd]: You're right, it did make use of option is findable, it seems declaring the comfy_pillow_trigger on ready makes it start fine and all symbols work with adjacents

**Comment by kazyofloof** [comment]
Turns out, re-adding that block with the fix has made comfy pillow non-functional, while it worked with the former method. Hmph.

**Pull request merged**